feat: updated corejs to latest stable, 2 not longer mantained

semoal opened the pull request.

Commit:
feat: updated corejs to latest stable, 2 not longer mantained

semoal commented:
I let the type of corejs with 2 | 3, if someone still wants to use 2..

DanielSchaffer commented:
@semoal thanks for starting this. I'm going to try and take a look this weekend to make sure the switch to 3 is as simple as just the changes in this PR, because I'd started this upgrade before and I thought I remembered there being some other issues.

Since it's upgrading a major version on a core dependency by default (no way to be sure, but I think most folks aren't being explicit about the corejs version), I think it's fair to also call this a major version bump for this plugin as well, so I just want to be careful and make sure nothing's being completely broken.

Commit:
chore: updated yarn lock and tgz to offline mirror

Commit:
chore: fixed eslint

aaarichter commented on src/defaults.ts:
```diff
@@ -5,7 +5,7 @@ import { StandardTargetOptionsMap } from './babel.target.options'
   modules: false,
   useBuiltIns: 'usage',
-  corejs: 2,
+  corejs: 3,
```
https://github.com/zloirock/core-js#babelpreset-env mentions in the warning that one should specify both major-minor version, otherwise feature added in minor versions are not injected. (TBH thats an odd behaviour 😩 )

semoal replied:
Oh, that's pretty new that warning. 
I'll add the latest minor asap

Commit:
chore: fixed corejs prefers minor version

DanielSchaffer committed:
chore: Merge branch 'develop' into update-corejs-stable

DanielSchaffer committed:
chore(examples/react-example): add missing offline mirror files

DanielSchaffer committed:
chore: fix missing `=` in README

DanielSchaffer committed:
feat(examples/react): finish react example implementation
- fix react example build
- add example implementation to collect clicks and generate the example spec UI
- rename to examples/react (from examples/react-example)
